netlify/functions/feedback-management.js

```diff
@@ -0,0 +1,130 @@
+const { WebClient } = require('@slack/web-api');
+const querystring = require('node:querystring');
+const { JWT } = require('google-auth-library');
+const { sheets } = require('@googleapis/sheets');
+
+const slackChannelIds = {
+  'mui-x': 'C04U3R2V9UK',
+  'mui-core': 'C041SDSF32L',
+};
+
+const spreadSheetsIds = {
+  forLater: '1NAUTsIcReVylWPby5K0omXWZpgjd9bjxE8V2J-dwPyc',
+};
+
+// Slack API
+const slackClient = new WebClient(process.env.SLACK_BOT_TOKEN);
+
+/**
+ * @param {object} event
+ * @param {object} context
+ */
+exports.handler = async (event) => {
+  try {
```

Comment:
Should we validate these events a bit more, e.g. make sure they are POST requests?

Reply:
For now, it's not planned to add more actions, so I do not plan to distinguish between GET/POST/PULL, ... Is their another reason than semantic behavior?

Reply:
> Is their another reason than semantic behavior?

Yes, you at least need to refuse GET/HEAD requests to avoid someone crafting a link for this request. Links can be followed without user interaction, e.g. bots, retries, prefetches,...